src/components/Pagination/Pager.tsx

```diff
@@ -9,17 +9,10 @@ import { mergeClasses } from '../../shared/utilities';
 
 import styles from './pagination.module.scss';
 
-/** Represents the number of pages from each edge of the list before we show quick buttons. */
-const EDGE_BUFFER_THRESHOLD: number = 5;
-
-/** Represents a list too short to display meaningful quick buttons. */
-const SHORT_LIST_THRESHOLD: number = 10;
-
 /** Represents the number of list items (pages) visible at any given time. */
 const VISIBLE_PAGER_COUNT = {
-  [PaginationVisiblePagerCountSizeOptions.Small]: 3,
-  [PaginationVisiblePagerCountSizeOptions.Medium]: 5,
-  [PaginationVisiblePagerCountSizeOptions.Large]: 7,
```

Comment:
This is a breaking change, you'll need to deprecate this as opposed to deleting.

Reply:
One way to avoid deprecating the prop is to change the value returned without removing one of the enum keys that map to your static:

```
const VISIBLE_PAGER_COUNT = {
  [PaginationVisiblePagerCountSizeOptions.Small]: 5,
  [PaginationVisiblePagerCountSizeOptions.Medium]: 7,
  [PaginationVisiblePagerCountSizeOptions.Large]: 7, // TODO: Remove in Octuple v3.0.0
};
```

Reply:
Added a pagerSize prop instead